drivers/display/display_st7789v.c

```diff
@@ -60,23 +61,22 @@ struct st7789v_data {
 #define ST7789V_PIXEL_SIZE 3u
 #endif
 
-static void st7789v_set_lcd_margins(const struct device *dev,
-				    uint16_t x_offset, uint16_t y_offset)
+static void st7789v_set_lcd_margins(const struct device *dev, uint16_t x_offset, uint16_t y_offset)
 {
 	struct st7789v_data *data = dev->data;
 
 	data->x_offset = x_offset;
 	data->y_offset = y_offset;
 }
 
-static void st7789v_transmit(const struct device *dev, uint8_t cmd,
-			     uint8_t *tx_data, size_t tx_count)
+static void st7789v_transmit(const struct device *dev, uint8_t cmd, uint8_t *tx_data,
+			     size_t tx_count)
 {
 	const struct st7789v_config *config = dev->config;
 	uint16_t data = cmd;
 
-	struct spi_buf tx_buf = { .buf = &cmd, .len = 1 };
-	struct spi_buf_set tx_bufs = { .buffers = &tx_buf, .count = 1 };
+	struct spi_buf tx_buf = {.buf = &cmd, .len = 1};
```

Comment:
Did you run `clang-format` on this driver file? I have no issue with these sorts of cleanups, but please put the clang-format changes into a separate commit- otherwise this sort of PR is difficult to review, and the git blame will be confusing as well :)

Reply:
Yes ran clang-format. Should've turned off format on save 😨 

For separate commits - do you want it as completely separate PRs or two commits (1 for reformat, 1 for the actual enhancement) in the same PR?

Reply:
Two in one PR is totally okay- I'd just like separate commits to make things easier to follow in the git history :)

Reply:
Fixed!